src/locales/messages.ts

```diff
@@ -1028,13 +1173,37 @@ export default defineMessages({
     description: 'View all {value}',
     id: 'DetailsViewAll',
   },
+  DiscountMinus: {
+    defaultMessage: 'EN Discount (-)',
+    description: 'Discount (-)',
+    id: 'DiscountMinus',
+  },
+  DistributionModelDesc: {
+    defaultMessage:
+      'EN This choice is for users to direct how their raw costs are distributed either by CPU or Memory on the project level breakdowns.',
+    description:
+      'This choice is for users to direct how their raw costs are distributed either by CPU or Memory on the project level breakdowns.',
+    id: 'DistributionModelDesc',
+  },
+  DistributionType: {
+    defaultMessage: 'EN Distribution type',
+    description: 'Distribution type',
+    id: 'DistributionType',
+  },
```

Comment:
This is a dup of CostModelsDistributionType.

Thinking CostModelsDistributionType should be replaced with this one.